resources/assets/lib/stores/channel-store.ts

```diff
@@ -200,9 +218,15 @@ export default class ChannelStore implements DispatchListener {
   private handleChatMessageNewEvent(event: MessageNewEvent) {
     for (const message of event.json.messages) {
       const channel = this.channels.get(message.channel_id);
-      if (channel == null) continue;
 
-      channel.addMessage(message);
+      if (channel != null) {
+        channel.addMessage(message);
+      } else if (!this.ignoredChannels.has(message.channel_id)) {
+        getChannel(message.channel_id)
```

Comment:
because the check for this endpoint is using auth check instead of userChannel existence, this causes weird behavior for admin account 🫠